src/audio/codec_adapter/codec/cadence.c

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
+//
+// Author: Marcin Rajwa <marcin.rajwa@linux.intel.com>
+
+/*
+ * \file cadence.c
+ * \brief Cadence Codec API
+ * \author Marcin Rajwa <marcin.rajwa@linux.intel.com>
+ *
+ */
+
+#include <sof/audio/codec_adapter/codec/generic.h>
+#include <sof/audio/codec_adapter/codec/cadence.h>
+
+int cadence_codec_init(struct comp_dev *dev)
+{
```

Comment:
In the Linux kernel I noticed that most of the functions written for Cadence code use cdns instead of Cadence. Maybe this is to have shorter lines?

So, I think it might worth having a look at this. Instead of cadence_codec_init -> cdns_codec_init.

Reply:
When the Cadence driver was made the line length limit was still 80, so there's that.

Reply:
@paulstelian97 OK, no hard feelings on this :). We can use cadence instead of cdns :)